puppet/modules/quickstack/manifests/pacemaker/galera.pp

```diff
@@ -170,9 +170,6 @@
     }
 
     exec {"pcs-mysqlinit-server-setup":
-      command => "/usr/sbin/pcs property set mysqlinit=running --force",
-    } ->
-    exec {"pcs-mysqlinit-server-set-up-on-this-node":
       command => "/tmp/ha-all-in-one-util.bash update_my_node_property mysqlinit"
```

Comment:
Doesn't matter.  The old line 173 wasn't setting a property that was referenced anywhere and was just a pointless extra step in the flow of execution.